conf/messages.en

```diff
@@ -1480,3 +1480,10 @@ evcpHowMuchDoYouWantToClaim.error.required.agent = Select if your client wants t
 evcpHowMuchDoYouWantToClaim.error.required.amount = Enter an amount up to the full cost spent on electric vehicle charge-points
 evcpHowMuchDoYouWantToClaim.error.overMax = The amount must be less than the full cost spent on electric vehicle charge-points
 evcpHowMuchDoYouWantToClaim.change.hidden = Change how much of the cost of electric vehicle charge-points should be claimed under the 100% allowance
+
+########## Capital Allowances - Balancing Allowance Amount ##########
+
+balancingAllowanceAmount.title.individual = How much is the total of your balancing allowances?
+balancingAllowanceAmount.title.agent = How much is the total of your client’s balancing allowances?
+balancingAllowanceAmount.error.required.individual = Enter the amount you spent on balancing allowance
```

Comment:
According to the document, these should be
'Enter the amount of the balancing allowance you want to report '
'Enter the amount of the balancing allowance your client wants to report'